demos/demo_simulate_nyu_finger_double_separate.py

```diff
@@ -0,0 +1,69 @@
+#!/usr/bin/env python
+
+"""demo_simulate_nyu_finger_double
+
+Simple demo showing how the simulation setup works.
+
+License: BSD 3-Clause License
+Copyright (C) 2018-2021, New York University , Max Planck Gesellschaft
+Copyright note valid unless otherwise stated in individual files.
+All rights reserved.
+"""
+
+import time
+from robot_properties_nyu_finger.wrapper import NYUFingerRobot
+from bullet_utils.env import BulletEnv
+import numpy as np
+
+if __name__ == "__main__":
+
+    # ! Create a Pybullet simulation environment before any robots !
+    env = BulletEnv()
+
+    # Create a robot instance. This adds the robot to the simulator as well.
+    finger0 = NYUFingerRobot(finger_index=0)
+    finger1 = NYUFingerRobot(finger_index=1)
```

Comment:
The use of `finger_index` won't scale when we use a two finger and three finger setup. Therefore, how about we pass int he config object directly instead of an index?

Reply:
And please pass in an **Object** not a Class.